File: MyBackend/ModAuxNomina.Models/Anticipos/AnticipoHistorial.cs
namespace MyBackend.ModAuxNomina.Models.Anticipos
{
    public class AnticipoHistorial
    {
        public int idAnticipo { get; set; }
        public int idAmortizacion { get; set; }
        public int? anno { get; set; }
        public int? mes { get; set; }
        public DateTime? fechaD { get; set; }
        public string? fecha { get; set; }
        public decimal? importe { get; set; }
        public string? observaciones { get; set; }
        public int? tipoAmortizacion { get; set; }
        public string? desTipoAmortizacion { get; set; }
    }
}

File: MyBackend/MyFrontend-Angular/src/app/models/anticipos/anticipo-historial.ts
export interface AnticipoHistorial {
	idAnticipo: number;
	idAmortizacion: number;
	anno?: number;
	mes?: number;
	fechaD?: Date;
	fecha?: string;
	importe?: number;
	observaciones?: string;
	tipoAmortizacion?: number;
	desTipoAmortizacion?: string; 
}